.github/workflows/ci.yaml

```diff
@@ -9,15 +9,15 @@ on:
 
 jobs:
 
-  "sanity-Tests":
+  "validity-Tests":
     runs-on: macOS-latest
     steps:
     - name: Checkout
       uses: actions/checkout@v2
     - name: Install swiftformat
       run: brew install swiftformat
-    - name: Run sanity
-      run: ./scripts/sanity.sh .
```

Comment:
I'm probably missing something here. Can you give me a hint, why we should rename the script? I chose the name based on Apple's repos.

Reply:
@gwynne via slack: 

> It's yet another one of the examples of terminology which has been more commonly recognized than in previous times as being misused, offensive, or otherwise questionable - in this case, as a form of ableism towards mental illness.